core-bundle/contao/templates/backend/be_ace.html5

```diff
@@ -37,8 +37,9 @@ if ($GLOBALS['TL_CONFIG']['useCE']):
         }
       });
 
-      // Enable code autocompletion
       editor.setOptions({
+        enableKeyboardAccessibility: true,
+        // Enable code autocompletion
```

Comment:
Is this "correct" here? Then we have to revert that:

https://github.com/contao/contao/blob/455fb5af5069e945992a9880a4d3b83a15eb9451/core-bundle/assets/controllers/ace-controller.js#L20

Reply:
We should keep it in the be_ace so people can disable it if they really want to